meteor/server/migration/0_25_0.ts

```diff
@@ -98,18 +97,18 @@ export const addSteps = addMigrationSteps('0.25.0', [
 		},
 	},
 
-	renamePropertiesInCollection(
-		'asRunLog',
-		AsRunLog,
-		'AsRunLog',
-		{
-			rundownId: 'runningOrderId',
-			// segmentId:	'segmentId',
-			partInstanceId: 'segmentLineId',
-			pieceInstanceId: 'segmentLineItemId',
-		},
-		'migrateDatabaseCollections'
-	),
+	// renamePropertiesInCollection(
```

Comment:
I'm missing a migration in x_x_x.ts which clears up the no-longer-used asRunLog collection. See how we did it in R25: https://github.com/nrkno/tv-automation-server-core/blob/master/meteor/server/migration/deprecatedDatabases/0_25_0.ts#L18